### DISCUSSION TITLE
[question] installation mangosbot-bots

### DISCUSSION THREAD

**Discussion by xBarselonx** [celguar/mangosbot-bots#150] [Q&A] [answered]
hello celguar!
I am using https://github.com/cmangos/mangos-tbc server. How do I install your bots (https://github.com/celguar/mangosbot-bots) on this server.
I can get a little setup guide from you.

**Comment by mostlikely4r** [comment, marked as answer]
Hey xBarselonx,

As mentioned in the readme of the  bots (https://github.com/celguar/mangosbot-bots) you need a special fork (like https://github.com/celguar/mangos-tbc) to properly utilize the bot-module.

Personally I'd simply grab the full installer from (https://github.com/celguar/spp-classics-cmangos/releases) and only build the above way if you really want to adjust code.

**Reply by xBarselonx** [reply]
Is this version of Wow TBC 2.4.3 (https://github.com/celguar/spp-classics-cmangos/releases) based on CMaNGOS?

**Reply by mostlikely4r** [reply]
Correct. A fork of cmangos with some added modules. See: https://github.com/celguar/mangos-tbc
